src/pages/patterns/login-pattern/index.mdx

```diff
@@ -59,104 +55,290 @@ The login page is presented to users in the following scenarios:
+
+This type of login is often paired with a solid color background or a subsystem approved background texture. Because the form is the focal point, complex illustrations are not appropriate in this situation.
+
+<Row>
+<Column colLg={8}>
+
+![Example of centered login forms paired with a background texture](./images/10_Login_centered_736.png)
+
+<Caption>
+  Example of centered login forms paired with a background texture
+</Caption>
+
+</Column>
+</Row>
+
+#### Split-screen layout
+
+The split-screen page is an alternate design that can be used to include some marketing content or other visual treatment related to the product. The login portion of this layout uses the same design and behavior as the centered layout but is confined to one part of the page.
+
 Any additional content on this page should be minimal and easy to scan. It shouldn’t distract from the login form. The user’s primary goal is logging in and that should be reflected in the visual design and emphasis on the page.
 
+Subsytem approved background textures or illustrations are appropriate for use with the login form as long as the pairing is accessible and enhances the experience. When choosing colors for your illustrations, consider their association with your particular product or communication. Lean on IBM Design Language layout principles along with the type scale to achieve a clear hierarchy.
+
+Please note that the fluid login form can also be side-aligned to create space for marketing content or an illustration.
+
 <Row>
-<Column colLg={12}>
+<Column colLg={8}>
 
-![Login with marketing content](/images/Login_marketing_material.png)
+![Example of split-screen login forms paired with an illustration](./images/11_Login_positions_736.png)
 
-<Caption>Example of a split-screen login page with marketing content</Caption>
+<Caption>
+  Example of split-screen login forms paired with an illustration
+</Caption>
 
 </Column>
 </Row>
 
 There may be a need to include some marketing content on this page. When including additional content, be sure to keep the marketing and login content separated. Testing has shown that users don’t look outside of the login region to find related actions such as create account or SSO buttons, and often miss those actions if they are embedded in the marketing content.
 
-### Best practices
+<Row>
+<Column colLg={12}>
+
+![Example of split-screen login form paired with marketing content](./images/12_Login_split_1120.png)
+
+<Caption>
+  Keep marketing content, including links and CTAs separate from the login
+</Caption>
+
+</Column>
+</Row>
+
+### Spacing
+
+Since login forms can apear as the central focus of the screen or in conjunction with markting content in a split-screen layout. Margins and vertical spacing can vary according to context.
```

Comment:
```suggestion 
Since login forms can appear as the central focus of the screen or in conjunction with marketing content in a split-screen layout, margins and vertical spacing can vary according to context.
```